src/lib/validators.ts

```diff
@@ -47,6 +47,21 @@ export async function validateFilePath(
 	}
 }
 
+export async function validatePath(
+	path: string,
+	raise = true,
+): Promise<boolean> {
+	try {
+		await statAsync(path);
+		return true;
+	} catch (error) {
+		if (raise) {
+			throw error;
+		}
+		return false;
```

Comment:
Same as above, maybe throw or no return.

Reply:
Validators are dropped, as we are now using schema everywhere.